### PR TITLE
Task/271 redesign login screen

### PR DESCRIPTION
**Pull request by TomasPetrik**
Re-design current login screen by design on:

https://www.figma.com/file/JwEHS4hCfJvzBf5ZNXkg7H/Simplio-UI?node-id=384%3A11156

This screen will be only in dark mode.

### CODE REVIEW

File: lib/view/screens/password_reset_screen.dart [hunk 1]
```diff
@@ -22,210 +23,213 @@ class PasswordResetScreen extends StatelessWidget with PopupDialogMixin {
...
-                            border: const UnderlineInputBorder(
-                              borderSide:
-                                  BorderSide(color: SioColorsDark.black),
+                          ),
+                          Gaps.gap32,
+                          ThemedTextFormField(
+                            key: const Key('reset-screen-email-text-field'),
+                            autofocus: true,
+                            /*
+                            validator: (email) =>
```
Comment (jvrastil) [inline lib/view/screens/password_reset_screen.dart:130]: remove commented code here please

File: lib/view/screens/password_reset_screen.dart [hunk 2]
```diff
@@ -22,210 +23,213 @@ class PasswordResetScreen extends StatelessWidget with PopupDialogMixin {
...
+                              errorMessage:
+                                  context.locale.common_email_validation_error,
                             ),
-                            focusedBorder: const UnderlineInputBorder(
-                              borderSide:
-                                  BorderSide(color: SioColorsDark.whiteBlue),
+                            */
+                            style:
+                                const TextStyle(color: SioColorsDark.whiteBlue),
+                            decoration: InputDecoration(
```
Comment (jvrastil) [inline lib/view/screens/password_reset_screen.dart:139]: this is used on multiple places, right? Can we refactor it and reuse the code?

File: lib/view/screens/sign_in_screen.dart [hunk 3]
```diff
@@ -38,199 +41,261 @@ class SignInScreen extends StatelessWidget {
...
+                                      .state
+                                      .login
+                                      .emailValidator(
+                                        email,
+                                        errorMessage: context.locale
+                                            .common_email_validation_error,
+                                      ),
+                                  style: const TextStyle(
+                                      color: SioColorsDark.whiteBlue),
+                                  decoration: InputDecoration(
```
Comment (jvrastil) [inline lib/view/screens/sign_in_screen.dart:79]: same here

**Review by jvrastil — CHANGES_REQUESTED** [review]

File: lib/view/screens/password_reset_screen.dart (hunk 1 above)
Comment (TomasPetrik) [reply lib/view/screens/password_reset_screen.dart:130]: Ok, removed.

File: lib/view/screens/password_reset_screen.dart (hunk 2 above)
Comment (TomasPetrik) [reply lib/view/screens/password_reset_screen.dart:139]: Yes, that's good point. Refactored and using UnderLinedTextFormFieldDecoration on 3 places:
sign in input field, sign in password field and reset password input field.

File: lib/view/screens/sign_in_screen.dart (hunk 3 above)
Comment (TomasPetrik) [reply lib/view/screens/sign_in_screen.dart:79]: Refactored.

**Review by jvrastil — APPROVED** [review]

**Pull request merged**